### PR TITLE
Release 1.4.6

### PR DESCRIPTION
**Pull request by ignasi35**
- [x] draft release notes
 - [x] fix contributors count on release notes
 - [x] update version on docs page

### CODE REVIEW

File: src/blog/lagom-1.4.6.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+This release provides several bug fixes and updates.
+
+## Highlights:
+
+- Updates Akka to [2.5.13](https://akka.io/blog/news/2018/05/31/akka-2.5.13-released)
+- Updates Akka Persistence JDBC to [3.4.0](https://github.com/lagom/lagom/issues/1374)
+- Fix in testkit to [allow tests using Play's DBAPI](https://github.com/lagom/lagom/pull/1355)
+- Small fixes improving the error reporting for developers. E.g. improved error message in [some issues in the Service.Descriptor](https://github.com/lagom/lagom/issues/1362) and improved error message when there's [no connection settings for userland read-side DB](https://github.com/lagom/lagom/pull/1372)
+
+See GitHub for the [full list of commits](https://github.com/lagom/lagom/compare/1.4.5...1.4.6) and the issues and pull requests assigned to the [1.4.5 milestone](https://github.com/lagom/lagom/milestone/31?closed=1).
```
Comment (TimMoore) [inline src/blog/lagom-1.4.6.md]: Should say "1.4.6 milestone" (the link is correct).

Comment (ignasi35) [reply src/blog/lagom-1.4.6.md]: Thanks!

**Comment by ignasi35** [comment]
Please do not merge yet.

File: src/blog/lagom-1.4.6.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+## Highlights:
+
+- Updates Akka to [2.5.13](https://akka.io/blog/news/INVALID_URL)
+- Updates Akka Persistence JDBC to [3.4.0](https://github.com/lagom/lagom/issues/1374)
+- Fix in testkit to [allow tests using Play's DBAPI](https://github.com/lagom/lagom/pull/1355)
+- Small fixes improving the error reporting for developers. E.g. improved error message in [some issues in the Service.Descriptor](https://github.com/lagom/lagom/issues/1362) and improved error message when there's [no connection settings for userland read-side DB](https://github.com/lagom/lagom/pull/1372)
+
+See GitHub for the [full list of commits](https://github.com/lagom/lagom/compare/1.4.5...1.4.6) and the issues and pull requests assigned to the [1.4.6 milestone](https://github.com/lagom/lagom/milestone/31?closed=1).
+
+## Building GDPR capable systems with Lagom
```
Comment (TimMoore) [inline src/blog/lagom-1.4.6.md]: Let's remove this section for now so we can go ahead and publish this.

File: src/blog/lagom-1.4.6.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    The Lagom team has released Lagom 1.4.6
+---
+
+The Lagom team has released Lagom 1.4.6.
+
+This release provides several bug fixes and updates.
+
+## Highlights:
+
+- Updates Akka to [2.5.13](https://akka.io/blog/news/INVALID_URL)
```
Comment (TimMoore) [inline src/blog/lagom-1.4.6.md]: Akka will be publishing their release notes shortly. Then we can fix the link and publish.

**Comment by ignasi35** [comment]
Thanks @TimMoore for handling this in my absence!

**Pull request merged**